enhancements/network/upgrades-with-ovn-ic.md

```diff
@@ -0,0 +1,245 @@
+- all new multizone components: `ovnkube-node.yaml -> ../multi-zone-interconnect/ovnkube-node.yaml`,  `ovnkube-control-plane.yaml -> ../multi-zone-interconnect/ovnkube-control-plane.yaml`.
+
+The remaining files are for alerts and service monitors.
+```text
+$ ls -l bindata/network/ovn-kubernetes/self-hosted/multi-zone-interconnect-tmp
+-rw-rw-r-- 1  7148 Aug 17 15:25 alert-rules-control-plane.yaml
+lrwxrwxrwx 1    53 Aug 17 15:25 monitor-control-plane.yaml -> ../multi-zone-interconnect/monitor-control-plane.yaml
+lrwxrwxrwx 1    47 Aug 17 15:25 monitor-master.yaml -> ../single-zone-interconnect/monitor-master.yaml
+lrwxrwxrwx 1    53 Aug 17 15:25 ovnkube-control-plane.yaml -> ../multi-zone-interconnect/ovnkube-control-plane.yaml
+lrwxrwxrwx 1    47 Aug 17 15:25 ovnkube-master.yaml -> ../single-zone-interconnect/ovnkube-master.yaml
+lrwxrwxrwx 1    44 Aug 17 15:25 ovnkube-node.yaml -> ../multi-zone-interconnect/ovnkube-node.yaml
+```
+
+CNO starts phase 2 by rolling out multizone ovnkube-node. Each node getting its ovnkube-node image updated will now run in its own zone with its own local ovnkube stack (ovnkube-controller, nbdb, northd, sbdb, ovn-controller) and will consider all other nodes as remote. Reversely, the rest of the cluster will consider this node as remote. 
+
+Once that is done, CNO will deploy the multizone ovnkube-control-plane deployment on master nodes. The ovnkube-control-plane pods only run ovnkube-cluster-manager: the three instances elect a leader, which will allocate IP subnets to nodes in a centralized manner.
+
+At this point of phase 2, the cluster runs:
+- multizone ovnkube-node (distributed per-node OVN stack)
+- multizone ovnkube-control-plane (slimmed-down centralized control plane)
+- single-zone ovnkube-master (centralized, but with no dataplane instances any longer listening to it)
+
+We're now in the final bit of phase 2, for which the YAMLs are to be found in `bindata/network/ovn-kubernetes/self-hosted/multi-zone-interconnect`: this is also the folder that is used for any fresh install of 4.14. The only remaining action is to remove the single-zone ovnkube-master pods, which are now of no use.
+
+```text
+$ ls -l bindata/network/ovn-kubernetes/self-hosted/multi-zone-interconnect/
+-rw-rw-r-- 1   7077 Aug 17 15:25 alert-rules-control-plane.yaml
+-rw-rw-r-- 1   1206 Aug 17 15:25 monitor-control-plane.yaml
+-rw-rw-r-- 1   5641 Aug 17 15:25 ovnkube-control-plane.yaml
+-rw-rw-r-- 1  52271 Sep  1 11:58 ovnkube-node.yaml
+```
+We have finally completed the whole upgrade from 4.13 to 4.14 and the cluster is now running:
+- multizone ovnkube-node
+- multizone ovnkube-control-plane
+
+CNO reports 4.14 in its operator status and CVO can carry on with the upgrade of the remaining operators.
```

Comment:
thanks Ric, this section was well written, nit: showing the oc get co network's output at each stage would have been awesome in letting ppl know when network operator has started, is doing, is done.

Reply:
Sure. My only concern is that the exact words in the log lines might change in the future and this part might get outdated soon...